src/test/testLogger.ts

```diff
@@ -40,12 +44,17 @@ export class TestLogger implements Logger {
             .map(loggedEntry => loggedEntry.entry)
     }
 
+    public writeLoggedEntriesToFile(entry: Loggable) {
```

Comment:
nit: `writeLoggedEntryToFile`

Reply:
Yep this was a copy paste of addLoggedEntries, fixed